core/src/main/scripts/importer/cbioportalImporter.py

```diff
@@ -410,6 +405,25 @@ def interface():
                         help='Path to Data file')
```

Comment:
all these commands above are shared among the various subcommands. I think the way to handle that in argparse is to add them to the parent command as described here: https://stackoverflow.com/questions/7498595/python-argparse-add-argument-to-multiple-subparsers